Imagetospeech/image_to_speech.py

```diff
@@ -0,0 +1,30 @@
+import pytesseract
+from PIL import Image
+from gtts import gTTS
+import os
+
+
+pytesseract.pytesseract.tesseract_cmd = input(r'Enter the path: ')
+
+img = input(r"Enter image: ")
```

Comment:
You resolved this conversation without modification:

![image](https://user-images.githubusercontent.com/43691873/113669205-06b83980-96d1-11eb-8be8-b65006443357.png)